### ISSUE TITLE
model training hyperparameter

### ISSUE THREAD

**Issue by Agino-ltp** [wilile26811249/MobileViT#1]
A problem has been bothering me. the learning rate, optimizer, batch_size, L2 regularization, label smoothing and epochs are inconsistent with the paper. How should I modify the code?

**Comment by wilile26811249** [comment]
Hi I am currently mainly implement the  model architecture. 

Regarding the training method mentioned in the paper, I will update it in the future.
